lambdas/data-migration2/tests/test-granulesAndFiles.js

```diff
@@ -0,0 +1,538 @@
+});
+
+test.after.always(async (t) => {
+  await granulesModel.deleteTable();
+  await collectionsModel.deleteTable();
+  await executionsModel.deleteTable();
+
+  await s3Utils.recursivelyDeleteS3Bucket(bucket);
+
+  await secretsManager().deleteSecret({
+    SecretId: process.env.cmr_password_secret_name,
+    ForceDeleteWithoutRecovery: true,
+  }).promise();
+
+  await destroyLocalTestDb({
+    ...t.context,
+    testDbName,
+  });
+});
+
+test.serial('migrateGranuleRecord correctly migrates granule record', async (t) => {
+  const {
+    collectionCumulusId,
+    executionCumulusId,
+    granulePgModel,
+    knex,
+    testGranule,
+  } = t.context;
+
+  await migrateGranuleRecord(testGranule, knex);
+
+  const record = await granulePgModel.get(knex, {
+    granule_id: testGranule.granuleId,
+    collection_cumulus_id: collectionCumulusId,
+  });
+
+  t.deepEqual(
+    omit(record, ['updated_at', 'cumulus_id']),
+    {
+      granule_id: testGranule.granuleId,
+      status: testGranule.status,
+      collection_cumulus_id: collectionCumulusId,
+      published: testGranule.published,
+      duration: testGranule.duration,
+      time_to_archive: testGranule.timeToArchive,
+      time_to_process: testGranule.timeToPreprocess,
+      product_volume: testGranule.productVolume.toString(),
+      error: testGranule.error,
+      cmr_link: testGranule.cmrLink,
+      execution_cumulus_id: executionCumulusId,
+      pdr_cumulus_id: null,
+      provider_cumulus_id: null,
+      query_fields: null,
+      beginning_date_time: new Date(testGranule.beginningDateTime),
+      ending_date_time: new Date(testGranule.endingDateTime),
+      last_update_date_time: new Date(testGranule.lastUpdateDateTime),
+      processing_end_date_time: new Date(testGranule.processingEndDateTime),
+      processing_start_date_time: new Date(testGranule.processingStartDateTime),
+      production_date_time: new Date(testGranule.productionDateTime),
+      timestamp: new Date(testGranule.timestamp),
+      created_at: new Date(testGranule.createdAt),
+    }
+  );
+});
+
+test.serial('migrateFileRecord correctly migrates file record', async (t) => {
+  const {
+    filePgModel,
+    knex,
+    testGranule,
+  } = t.context;
+
+  const testFile = testGranule.files[0];
+  await migrateGranuleRecord(testGranule, knex);
+  await migrateFileRecord(testFile, testGranule.granuleId, testGranule.collectionId, knex);
+
+  // I am not sure how I can select a file where bucket and key are null
+  const record = await filePgModel.get(knex, {});
+
+  t.deepEqual(
+    omit(record, fileOmitList),
+    omit({
+      bucket: testFile.bucket ? testFile.bucket : null,
+      checksum_value: testFile.checksum,
+      checksum_type: testFile.checksumType,
+      key: testFile.key ? testFile.key : null,
+      path: null,
+      file_size: testFile.size.toString(),
+      file_name: testFile.fileName,
+      source: testFile.source,
+      type: testFile.type,
+    },
+    ['type'])
```

Comment:
Seems like we can just not add `type` to the object and then remove the `omit`?

Reply:
Yeah 🤦 [`cc2dabc` (#2051)](https://github.com/nasa/cumulus/pull/2051/commits/cc2dabc6a12ab1c87781492c881a5b21a5b51ebb)